build.gradle

```diff
@@ -151,57 +183,33 @@ task createPackages(description: 'Create three packages for Linux, Windows and M
     //This task also depends on other three tasks which are created by iterating osNames
 }
 
-osNames.each {String sys ->
-    def packageDir = new File("$buildDir/distributions/packages")
-    def (taskType, fileExtension, fileCompression, scriptFile) = setPackageProperties(sys)
-    def taskCLass = Class.forName(taskType)
-    def jarDir = file("$buildDir/libs")
-    def scriptDir = file("$projectDir/launcher-scripts")
-    task "packageFor$sys"(type: taskCLass){
-        mustRunAfter unzipJre
-        archiveBaseName = "admin-console-package-for-$sys"
-        extension = fileExtension
-        if(fileCompression) compression = Compression.GZIP
-        destinationDirectory = packageDir
-        from jarDir
-        from file(jreExtractSubDir(sys))
-        exclude (['*.zip', '*.tar.gz'])
-        from file("${scriptDir.getAbsolutePath()}/$scriptFile")
-
-        doLast {
-            println("Package for $sys is created under ${packageDir.absolutePath}")
+packagePropertiesMap.each {String sys, PackageProperties props ->
```

Comment:
I'm aware you're reworking this area with the `distribution` plugin, so I'll wait for next batch...